.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- Make sure the `Allow edits and access to secrets by maintainers` checkbox is checked on this pull request. -->
+
+## Changes:
+
+<!-- Describe what behavior is changed by this PR. -->
+
+## Context:
+
+<!-- Describe why you're making these changes if it's not already explained in a corresponding issue. -->
+<!-- If you're closing an existing issue with this pull request, use the keyword Closes #issue_number -->
+
+## Documentation (please check one with an [x])
+
+- [ ] I have updated the documentation, or
+- [ ] No documentation update is required
+
```

Comment:
```suggestion
```

Maybe this could go as well? I'm not sure what the end-situation is going to be once the Docsuaurus v2 migration is done.